bin/docserv-createconfig

```diff
@@ -77,7 +79,6 @@ def get_version(name):
             return version
     return name
 
-
 def prettify(tree, indent="  "):
```

Comment:
I've introduced this function because the `xml` library didn't support pretty printing directly. With `lxml`, pretty printing can be done almost automatically. Basically you *could* replace all the code inside this function with this:

```python
def prettify(root):
    # ...
    return etree.tostring(root, pretty_print=True, encoding="unicode")
```

When you call this function, you need to pass just the root element, no the tree.

However, there is one difference: you can't specify the indentation. You can only specify if or if not you want to pretty print your tree. Not sure if this is useful.

Reply:
I find it helpful, thanks @tomschr!